content/blog/2022-07-26-schema-evolution-glue-msk/index.md

```diff
@@ -0,0 +1,861 @@
+---
+title: Schema Evolution with Glue Schema Registry and Managed Streaming for Kafka (MSK) using LocalStack
+description: Find incompatibilities early or even avoid them alltogether when developing Kafka producers or consumers! Learn how to test data schema evolution by using Managed Streaming for Kafka (MSK) with the Glue Schema Registry in LocalStack.
+lead: Find incompatibilities early or even avoid them alltogether when developing Kafka producers or consumers! Learn how to test data schema evolution by using Managed Streaming for Kafka (MSK) with the Glue Schema Registry in LocalStack.
+images: ["schema-evolution-glue-msk-poster.png"]
+contributors: ["Alexander Rashed"]
+date: 2022-07-26
+tags: ['tutorial']
+leadimage: "schema-evolution-glue-msk-poster.png"
+resources:
+- src: "schema-evolution-glue-msk-flow.svg"
+---
+
+{{< img src="schema-evolution-glue-msk-poster.png" >}}
+
+[Apache Kafka](https://kafka.apache.org/) is an open-source distributed event store and stream-processing platform.
+It is used to capture data generated by producers and distribute it among its consumers.
+Kafka is known for its scalability, with reports of production environments scaling to [trillions of messages per day](https://engineering.linkedin.com/blog/2019/apache-kafka-trillion-messages).
+With [Amazon Managed Streaming for Apache Kafka (MSK)](https://aws.amazon.com/msk/), AWS provides a service to easily provision Apache Kafka clusters.
+[LocalStack Pro supports Amazon Managed Streaming for Kafka (MSK)](https://docs.localstack.cloud/aws/managed-streaming-for-kafka/), which enables you to spin up Kafka clusters on your local machine and testing the integration of your applications with Amazon MSK.
+
+Kafka clusters are often used as the central messaging infrastructure in complex microservice environments.
+However, the continuous and independent development of the individual microservices - the data producers and consumers - can make it hard to coordinate and evolve data schemas over time without introducing application failures due to incompatibilities.
+A common solution to this problem is to use a schema registry which provides for the validation of schema changes, preventing any unsafe changes and subsequent application failures.
```

Comment:
As far as I can see, this suggestion only contains the removal of newlines.
Since the markdown dialect used in Hugo doesn't render single new-lines, this wouldn't have an effect on the resulting page and makes diffs a bit harder. Therefore I won't apply this change.